docs/articles/new-expensify/expensify-card/Disputing-Expensify-Card-Transactions.md

```diff
@@ -37,45 +37,67 @@ If you encounter unrecognized, unauthorized, or fraudulent charges on your Expen
 5. Follow the prompts to deactivate your card and request a replacement.
 
 ## 2. Contact Expensify
-   - After deactivating your card, contact Expensify Concierge to report the fraudulent transaction:
-     - **Expensify Chat:** Open your Expensify Chat inbox and message Concierge.
-     - **Email:** Send details to concierge@expensify.com.
-   - Provide the following information:
-     - Details of the fraudulent charge (e.g., date, amount, merchant name).
-     - Any supporting documentation.
+
+After deactivating your card, contact Expensify Concierge to report the fraudulent transaction:
+
+- **Expensify Chat:** Open your Expensify Chat inbox and message Concierge
+- **Email:** Send details to concierge@expensify.com
+
+Provide the following information:
+- Details of the fraudulent charge (e.g., date, amount, merchant name).
+- Any supporting documentation.
 
 ## 3. Enable Two-Factor Authentication (2FA)
 
-If you haven't already, strengthen your account’s security by [enabling 2FA](https://help.expensify.com/articles/new-expensify/settings/Enable-Two-Factor-Authentication).
+If you haven't already, strengthen your account's security by [enabling 2FA](https://help.expensify.com/articles/new-expensify/settings/Enable-Two-Factor-Authentication).
 
 ---
 
 # Steps to Report a Service Dispute
 
-## Contact the Merchant
+## Contact the merchant
 
-Most service-related issues can be resolved directly with the merchant. This is the quickest way to address disputes.
+Most service-related issues can be resolved directly with the account. This is the quickest way to address disputes.
```

Comment:
> Most service-related issues can be resolved directly with the `account`

Did you mean to change this to `account`? I would think we would want to keep `merchant` here, right?